src/router/user.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response, Router } from 'express'
+import { ContainerTypes, createValidator, ValidatedRequest, ValidatedRequestSchema } from 'express-joi-validation'
+import { createUpdateSchema } from '../schema/user'
+import { User } from 'src/entity/user'
+
+const router:Router = express.Router()
+const validator = createValidator()
+
+// create and update user
+interface createSchema extends ValidatedRequestSchema {
+    [ContainerTypes.Body]: User
+}
+router.post('/createUpdateUser', validator.body(createUpdateSchema),
+    (req: ValidatedRequest<createSchema>, res: Response) => {
+    res.send("create and update user!")
```

Comment:
For all the API, store user’s collection in the service memory. We need an object, and CRUD with that object